config/prod.exs

```diff
@@ -14,9 +14,26 @@ use Mix.Config
 # manifest is generated by the mix phx.digest task
 # which you typically run after static files are built.
 config :elixir_bench, ElixirBenchWeb.Endpoint,
+  # Dynamically verify and set Endpoint env variables
   load_from_system_env: true,
+  http: [
+    protocol_options: [
+      max_header_name_length: 640,
+      max_header_value_length: 8192,
+      max_headers: 100
```

Comment:
do we need these?

Reply:
I just increased `max_header_value_lenght`, other values are default. My browser was issuing problems with big header cookies, so I doubled the allowed size of a header. I cleaned my cached later and it worked with the default value of 4096, but I don't think this config is too bad.